src/levelset/levelSetCommon.hpp

```diff
@@ -177,6 +177,15 @@ enum class LevelSetField{
     UNDEFINED
 };
 
+/*!
+ * @ingroup levelsetEnums
+ * Enum class containing the possible order of surface smooting
+ */
+enum class LevelSetSurfaceSmoothing{
+    LOW_ORDER,                      /**< level set high order surface smoothing */
+    HIGH_ORDER                      /**< level set low order surface smoothing */
```

Comment:
Add the HIGH_ORDER flag when high order smoothing is introduced.

Reply:
You can remove the "level set" text, just ```//! High order surface smoothing```. Also, the comments are swapped.

Reply:
I removed the "level set" text. I don't understand your first comment. Could you please rephrase it?

Reply:
> I removed the "level set" text. I don't understand your first comment. Could you please rephrase it?

You have added the HIGH_ORDER constant when the code doesn't yet support high order surface smoothing. The HIGH_ORDER constant should be introduced at the same time as the high order smoothing.

Reply:
Done